core/chainlink.Dockerfile

```diff
@@ -29,6 +29,8 @@ RUN make contracts-operator-ui-build
 FROM golang:1.17-buster
 WORKDIR /chainlink
 
+RUN wget https://github.com/CosmWasm/wasmvm/raw/main/api/libwasmvm.so -O /usr/lib/libwasmvm.so
```

Comment:
This should fetch from a specific tag that matches Terra, in our case 0.16. The latest branch is currently on 1.0.0 beta which might be incompatible

Reply:
Is this actually necessary? The library is used for running the CosmWasm VM

Reply:
Looks like you could also copy it straight from the go package: https://github.com/terra-money/core/blob/0b3a30669f817b58535c06f3c8eee48072e0ffde/shared.Dockerfile#L14

Reply:
:point_up: what @archseer said

Reply:
I like the idea of just pulling it from the go packageh